src/main/kotlin/snyk/common/lsp/LanguageServerWrapper.kt

```diff
@@ -405,9 +406,13 @@ class LanguageServerWrapper(
             integrationVersion = pluginInfo.integrationVersion,
             authenticationMethod = authMethod,
             enableSnykOSSQuickFixCodeActions = "true",
+            enableDeltaFindings = isDeltaFindingsEnabled(ps),
         )
     }
 
+    private fun isDeltaFindingsEnabled(ps: SnykApplicationSettingsStateService) =
+        (ps.netNewIssues == "Net new issues").toString()
+
```

Comment:
Why is this duplicated? Doesn't the definition in the settings already does the same?

Reply:
Here we set the var netNewIssues: in SnykApplicationSettingsStateService. I think it also needs to be done for the setting in LanguageServerSettings.

Reply:
100% Will fix!